File: app/src/main/java/com/aivoice/translate/MyApp.java
package com.aivoice.translate;

import android.app.Application;
import android.content.Context;

import com.google.android.gms.ads.MobileAds;
import com.google.android.gms.ads.initialization.InitializationStatus;
import com.google.android.gms.ads.initialization.OnInitializationCompleteListener;

public class MyApp extends Application {
    Context context;
    public static MyApp myApp;

    public static MyApp getInstance() {
        return myApp;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        context = this;
        myApp = this;

        MobileAds.initialize(
                this,
                new OnInitializationCompleteListener() {
                    @Override
                    public void onInitializationComplete(InitializationStatus initializationStatus) {}
                });
    }
}

File: app/src/main/java/com/texttranslate/voiceimage/MyApp.kt
package com.texttranslate.voiceimage

import android.annotation.SuppressLint
import android.app.Application
import android.content.Context

class MyApp : Application() {
    var context: Context? = null
    override fun onCreate() {
        super.onCreate()
        context = this
        instance = this
    }

    companion object {
        @SuppressLint("StaticFieldLeak")
        var instance: MyApp? = null
    }
}
